src/main/java/org/wmn4j/io/musicxml/MusicXmlWriterDom.java

```diff
@@ -719,6 +721,9 @@ private Element createNoteElement(Note note, int voiceNumber, Integer staffNumbe
 			noteElement.appendChild(createStaffElement(staffNumber));
 		}
 
+		// Notations (musical notations, e.g. articulations, dynamics, fermata, slur)
+		createNotationsElement(note).ifPresent(notationsElement -> noteElement.appendChild(notationsElement));
```

Comment:
Does this work also when there are multiple staves? I think there might be some order in which the child elements need to be inside a note element. I originally put the staff element before appearance and the outputted files failed validation and I think the order was the cause.